pandas/parser.pyx

```diff
@@ -1208,7 +1208,16 @@ cdef class TextReader:
             else:
                 return None
 
-class CParserError(Exception):
+
```

Comment:
what are you doing here? you are making things more complicated. now we have different kinds of errors from different parsers? how is that making things simpler